packages/admin/admin/src/stack/breadcrumbs/BreadcrumbsEntry.tsx

```diff
@@ -1,42 +1,89 @@
+import { css, styled, useThemeProps } from "@mui/material/styles";
+import { ThemedComponentBaseProps } from "helpers/ThemedComponentBaseProps";
 import * as React from "react";
 
 import { BreadcrumbItem } from "../Stack";
 import { BreadcrumbLink } from "./BreadcrumbLink";
-import { styles } from "./StackBreadcrumbs.styles";
 
-interface BreadcrumbsEntryProps {
+const StyledLink: typeof Link = styled(Link, {
+    name: "CometAdminStackBreadcrumbs",
+    slot: "link",
+    overridesResolver(_, styles) {
+        return [styles.link];
+    },
+})(
+    ({ theme }) => css`
+        font-size: 13px;
+        line-height: 14px;
+        font-weight: ${theme.typography.fontWeightMedium};
+        color: ${theme.palette.grey[600]};
+        text-decoration-color: currentColor;
+    `,
+);
+
+const StyledDisabledLink = styled(Typography, {
+    name: "CometAdminStackBreadcrumbs",
+    slot: "disabledLink",
+    overridesResolver(_, styles) {
+        return [styles.disabledLink];
+    },
+})(
+    ({ theme }) => css`
+        font-size: 13px;
+        line-height: 14px;
+        font-weight: ${theme.typography.fontWeightMedium};
+        color: ${theme.palette.grey[600]};
+        text-decoration-color: currentColor;
+        color: ${theme.palette.text.disabled};
+    `,
+);
+
+const BackButtonSeparator = styled("div", {
+    name: "CometAdminStackBreadcrumbs",
+    slot: "backButtonSeparator",
+    overridesResolver(_, styles) {
+        return [styles.backButtonSeparator];
+    },
+})(
+    ({ theme }) => css`
+        height: 30px;
+        width: 1px;
+        background-color: ${theme.palette.divider};
+        margin-right: 12px;
+    `,
+);
+
+export interface BreadcrumbsEntryProps
+    extends ThemedComponentBaseProps<{ link: typeof Link; backButtonSeparator: "div"; disabledLink: typeof Typography }> {
     item: BreadcrumbItem;
     isLastItem?: boolean;
     backButtonUrl?: string;
 }
 
-export const BreadcrumbsEntry = ({
-    item,
-    isLastItem,
-    backButtonUrl,
-    classes,
-}: BreadcrumbsEntryProps & WithStyles<typeof styles>): React.ReactElement => (
-    <>
-        {backButtonUrl && (
-            <>
-                <IconButton className={classes.backButton} component={BreadcrumbLink} to={backButtonUrl}>
-                    <LevelUp />
-                </IconButton>
-                <div className={classes.backButtonSeparator} />
-            </>
-        )}
-        {isLastItem ? (
-            <Typography className={clsx(classes.link, classes.disabledLink)} variant="body2">
-                {item.title}
-            </Typography>
-        ) : (
-            <Link to={item.url} component={BreadcrumbLink} className={classes.link} variant="body2">
-                {item.title}
-            </Link>
-        )}
-    </>
-);
+export function BreadcrumbsEntry(inProps: BreadcrumbsEntryProps) {
+    const { item, isLastItem, backButtonUrl, slotProps } = useThemeProps({ props: inProps, name: "CometAdminBreadcrumbsEntry" });
```

Comment:
`ThemedComponentBaseProps`, its slot-types and `useThemeProps` should only be defined/used in the "primary" component `StackBreadcrumbs` as `BreadcrumbsEntry` is not an exported component but only used internally by `StackBreadcrumbs`.